src/ErrorMessage.js

```diff
@@ -0,0 +1,8 @@
+const errorMessage = {
+    NO_PLAYER: '[ERROR]: 주어진 플레이어가 없습니다.',
+    NEGATIVE_NUMBER: '[ERROR]: 플레이어 수가 음수로 주어졌습니다.',
+    NAME_IS_TOO_LONG: '[ERROR]: 플레이어 이름은 5자를 초과할 수 없습니다.',
+    NOT_A_NUMBER: '[ERROR]: 플레이어 수 입력이 숫자가 아닙니다.'
+};

```

Comment:
```suggestion
const ERROR_MESSAGES = {
    NO_PLAYER: '[ERROR]: 주어진 플레이어가 없습니다.',
    NEGATIVE_NUMBER: '[ERROR]: 플레이어 수가 음수로 주어졌습니다.',
    NAME_IS_TOO_LONG: '[ERROR]: 플레이어 이름은 5자를 초과할 수 없습니다.',
    NOT_A_NUMBER: '[ERROR]: 플레이어 수 입력이 숫자가 아닙니다.'
};
```
여기도 상수 네이밍 컨벤션인 스네이크 케이스로 작성하는건 어떨까요??